api/db-cdr/csv/db_domain.csv

```diff
@@ -2,14 +2,14 @@
 "Condition","Diagnoses","Conditions are records of a Person suggesting the presence of a disease or medical condition stated as a diagnosis, a sign or a symptom, which is either observed by a Provider or reported by the patient.","domain_filter","condition",19,0
 "CPT4","CPT4","CPT or Current Procedural Terminology (CPT) is a product of the American Medical Association (AMA). CPT codes are used by health care professionals for billing of medical services and procedures to public and private health insurance programs. CPT is a list of descriptive terms and identifying numeric codes for medical services and procedures that are provided by physicians and health care professionals.","standard","cpt4",44819100,0
 "Drug","Medications","Drugs biochemical substance formulated in such a way that when administered to a Person it will exert a certain physiological or biochemical effect. The drug exposure domain concepts capture records about the utilization of a Drug when ingested or otherwise introduced into the body.","domain_filter","drug",13,0
-"Fam Med His","Family Medical History","The Family module provides information on family medical history","survey","PPI",NULL,0
+"Fam Med His","Family Medical History","The Family module provides information on family medical history","survey","PPI",0,0
```

Comment:
does this break any api , going from null to 0 ?  It is fine to have it 0 as default. Just make sure our api is not looking for null.

Reply:
Changed the api to fetch the domains that look out for 0 instead of null.